pghoard/create_keys.py

```diff
@@ -62,7 +62,9 @@ def show_key_config(site, key_id, rsa_private_key, rsa_public_key):
 
 
 def save_keys(config_file, site, key_id, rsa_private_key, rsa_public_key):
-    config_obj = config.read_json_config_file(config_file, check_commands=False)
+    with open(config_file) as fp:
```

Comment:
we won't get nice error messages about invalid / missing config any more.  a workaround would be to call the read_json_config_file but ignore its results and read the file like this if it didn't throw an error.  or edit the function to allow an optional flag to disable defaults